src/node/desktop/src/main/session-launcher.ts

```diff
@@ -558,6 +568,18 @@ export class SessionLauncher {
       }
     }
 
+    // if we're running automation tests, set that up now
+    if (app.commandLine.hasSwitch('run-automation')) {
+      argList.push('--run-automation');
+      if (!app.isPackaged) {
```

Comment:
Do we need to do anything here for packaged builds of RStudio? Maybe just set the automation mode?

Reply:
Will the automation work on a packaged build without setting these environment variables?

Reply:
They should; in this scenario, we would rely on the fallback behavior of downloading the automation tests from GitHub appropriate to the GitHub commit that copy of RStudio was built from, and then run them in the regular way.